t/venom/Makefile

```diff
@@ -39,3 +39,7 @@ global_teardown: setup
 .PHONY: dot1x_eap_peap_no_teardown
 dot1x_eap_peap_no_teardown: setup global_config
 	./venom-wrapper.sh $(PFSERVERS_DIR)/dot1x_eap_peap
+
+.PHONY: backup_db_and_restore
+backup_db_and_restore: setup
+	./venom-wrapper.sh $(PFSERVERS_DIR)/backup_db_and_restore
```

Comment:
- put this make target after `dot1x_eap_peap` target not after `dot1x_eap_peap_no_teardown`
- call teardown test suite
- add a make target with `no_teardown` suffix
- add `backup_db_and_restore` to `test` suite target

Reply:
done